README.md

```diff
@@ -228,7 +227,7 @@ signed-manifest = {
 
 manifest = {
   metadata: manifest-metadata,
-  resources: [* [resource: resource-key, hashes: hashes]],
+  resource-hashes: {* hash-algorithm => [hash-value]},
```

Comment:
No, the list of hash values here is just a set. If the resource matches any of them, it's valid. I think https://github.com/dimich-g/webpackage/pull/46/files#diff-04c6e90faac2675aa89e2176d2eec7d8R316 says this.